internal/registration/api/inprocess/registration/registration.go

```diff
@@ -0,0 +1,76 @@
+package inprocess
+
+import (
+	"context"
+
+	"GoEdu/internal/registration/api/registration"
+	"GoEdu/internal/registration/internal/application"
+	"GoEdu/internal/registration/internal/domain"
+)
+
+var _ registration.RegistrationModuleFacade = (*RegistrationModuleFacade)(nil)
+
+type RegistrationModuleFacade struct {
+	registrationHandler application.CommandHandler
+	confirmationHandler application.CommandHandler
+}
+
+func NewRegistrationModuleFacade(
+	registrationHandler,
+	confirmationHandler application.CommandHandler,
+) *RegistrationModuleFacade {
+	return &RegistrationModuleFacade{
+		registrationHandler: registrationHandler,
+		confirmationHandler: confirmationHandler,
+	}
+}
+
+func CreateNewInProcessRegistrationModuleFacade() *RegistrationModuleFacade {
```

Comment:
You should create a `module.go` inside `GoEdu/internal/registration` package and build all the module dependencies in there for now. The example is in `GoEdu/internal/api/http` module.